dz_1/dz_1_1.py

```diff
@@ -0,0 +1,13 @@
+"""
+Задача 2: Найдите сумму цифр трехзначного числа.
+*Пример:*
+123 -> 6 (1 + 2 + 3)
+100 -> 1 (1 + 0 + 0) |
+"""
+
+number = input("Введите трехзначное число: ")
+
+if len(number) == 3:
+    print(int(number[0])+int(number[1])+int(number[2]))
```

Comment:
Верно, но не забывайте про пробелы.
```suggestion
    print(int(number[0]) + int(number[1]) + int(number[2]))
```